src/main/scala/viper/gobra/reporting/Source.scala

```diff
@@ -39,6 +42,9 @@ object Source {
       override lazy val origin: Option[Origin] = Some(src)
       override def vprMeta(node: internal.Node): (vpr.Position, vpr.Info, vpr.ErrorTrafo) =
         (vpr.TranslatedPosition(src.pos), Verifier.Info(pnode, node, src), vpr.NoTrafos)
+
+      def annotateOrigin(annotation: Annotation): Single =
+        new Single(pnode, new AnnotatedOrigin(src, annotation))
```

Comment:
`new` keyword shouldn't be necessary (at least the first one)

Reply:
Removed the first one